content/en/about-filecoin/faq.md

```diff
@@ -78,35 +84,35 @@ If you lost the data itself, then no, there's no way to recover that, and you wi
 
 ### Has Filecoin confirmed the use of the SDR algorithm? Is there any evidence of malicious construction?
 
-SDR is confirmed and used, and we have no evidence of malicious construction. The algorithm is also going through both internal and external security audits.
+SDR ([Stacked DRG PoRep](https://spec.filecoin.io/algorithms/porep-old/stacked_drg/#section-algorithms.porep-old.stacked_drg)) is confirmed and used, and we have no evidence of malicious construction. The algorithm is also going through both internal and external security audits.
 
 If you have any information about any potential security problem or malicious construction, reach out to our team at [security@filecoin.org](mailto:security@filecoin.org).
 
 ### How likely is it that the Filecoin protocol will switch to the NSE Proof-of-Replication construction later?
 
-About NSE: NSE is one of the best candidates for a proof upgrade, and teams are working on implementation. But there are other candidates too, which are promising as well. It may be that another algorithm ends up better than NSE -- we don't know yet. Proof upgrades will arrive after the mainnet launch and will coexist.
+About NSE: NSE (Native Storage Extension) is one of the best candidates for a proof upgrade, and teams are working on implementation. But there are other candidates too, which are promising as well. It may be that another algorithm ends up better than NSE -- we don't know yet. Proof upgrades will arrive after the mainnet launch and will coexist.
```

Comment:
> About NSE: NSE (Native Storage Extension)

Try to use the acronym _after_ you've told the reader what it stands for.

```suggestion
Native storage extension (NSE) is one of the best candidates for a proof upgrade, and teams are working on implementation. But there are other candidates too, which are promising as well. It may be that another algorithm ends up better than NSE -- we don't know yet. Proof upgrades will arrive after the mainnet launch and will coexist.
```